src/pynwb/io/base.py

```diff
@@ -29,6 +29,7 @@ def __init__(self, spec):
         self.map_spec('unit', data_spec.get_attribute('unit'))
         self.map_spec('resolution', data_spec.get_attribute('resolution'))
         self.map_spec('conversion', data_spec.get_attribute('conversion'))
+        self.map_spec('continuity', data_spec.get_attribute('continuity'))
```

Comment:
It seems strange that this line is not being hit by tests since it is being set in "tests/integration/hdf5/test_base.py" . May be worth checking why this line is reported as not covered.

Reply:
Code coverage is assessed using unit tests only. Integration tests will hit the IO layer.